docs/getting_started.md

```diff
@@ -60,42 +85,78 @@ First of all, prepare JSON files
 }
 ```
 
-Read [ipam](ipam.md) if you want to know meaning of each parameter.
+Then put the JSON to sabakan:
+```console
+$ sabactl ipam set -f ipam.json
+```
 
-- dhcp.json
+Read [ipam.md](ipam.md) for details.
+
+### <a name="dhcp" />Configure DHCP
+
+Prepare `dhcp.json` as follows:
 ```json
 {
-   "gateway-offset": 100,
-   "lease-minutes": 120
+   "gateway-offset": 1,
 }
 ```
 
-Read [dhcp](dhcp.md) if you want to know meaning of each parameter.
-
-Use `sabactl` to configure `sabakan`.
-
+Then put the JSON to sabakan:
 ```console
-$ sabactl ipam set -f ipam.json
 $ sabactl dhcp set -f dhcp.json
 ```
 
-Make sure current configuration.
+Read [dhcp.md](dhcp.md) for details.
+
+### <a name="upload" />Upload CoreOS Container Linux
+
+Download CoreOS PXE boot images:
+```console
+$ curl -o kernel -Lf http://stable.release.core-os.net/amd64-usr/current/coreos_production_pxe.vmlinuz
+$ curl -o initrd.gz -Lf http://stable.release.core-os.net/amd64-usr/current/coreos_production_pxe_image.cpio.gz
+```
 
+Upload them to sabakan as follows:
 ```console
-$ sabactl ipam get
-$ sabactl dhcp get
+$ sabactl images upload ID kernel initrd.gz
 ```
 
-Each output will be the same as [above JSON](#configuresabakan)
+### <a name="machines" />Register machines
 
-## <a name="netboot" /> Netboot
+Prepare `machines.json` as follows:
+```json
+[
+  {
+    "serial": "1234abcd",
+    "product": "Dell R640",
+    "datacenter": "tokyo1",
+    "rack": 0,
+    "role": "boot",
+    "bmc": {
+      "type": "IPMI-2.0",
+      "ipv4": "10.72.17.37"
+    }
+  },
+  {
+    // another machine
+  }
+]
+```
 
-**ToDo**
+Then put the JSON to sabakan:
+```console
+$ sabactl machines create -f machines.json
+```
 
-## <a name="test" /> Test
+Each object in the array is [`MachineSpec`](machine.md#machinespec-struct).
+Sabakan identifies physical servers by `serial`.
 
-**ToDo**
+Once machines are properly registered with sabakan, they can netboot
+CoreOS Container Linux using [UEFI HTTP Boot][HTTPBoot].
```

Comment:
Ignition should be prepared after all the above steps. Should it be just added a link of the "ignition.md"?